src/tmi/io_utils.py

```diff
@@ -0,0 +1,237 @@
+import os
+import pathlib
+import warnings
+
+
+def validate_paths(paths):
```

Comment:
are we completely scrapping the data_prefix functionality?

Reply:
I was thinking so, when I was swapping them, there was never an instance where `data_prefix` was set to `True`. Should we keep it in?

Reply:
Sounds fine to me if we're not that concerned about docker scope anymore.